util/reggen/multi_register.py

```diff
@@ -84,6 +105,22 @@ def __init__(self,
                                 .format(self.reg.name))
         self.name = self.reg.name
 
+        self.alias_target = None
```

Comment:
Isn't the `alias_target` stuff already handled in the base class?

Reply:
Are you referring to the `Register`? `MultiRegister` derives from `RegBase` and not from `Register`...

Reply:
You're completely right! I've forgotten how my own code works... Sorry!